stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -29,10 +29,8 @@ internal class PaymentOptionsViewModel(
     application = application
 ) {
     @VisibleForTesting
-    internal val _viewState = MutableLiveData<ViewState.PaymentOptions>(
-        ViewState.PaymentOptions.Ready
-    )
-    internal val viewState: LiveData<ViewState.PaymentOptions> = _viewState.distinctUntilChanged()
+    internal val _viewState = MutableLiveData(ViewState.PaymentOptionsReady)
+    internal val viewState: LiveData<ViewState.PaymentOptionsReady> = _viewState.distinctUntilChanged()
```

Comment:
This doesn't appear to be doing anything anymore. The default value is `ViewState.PaymentOptionsReady` and that's the only thing that will ever be emitted. Can we remove it?